### PR TITLE
Create course eval

### PR DESCRIPTION
**Pull request by MISTERTAC0H**
Added rating option in CourseEval.php

### CODE REVIEW

File: database/dbEvaluations.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+include_once('dbinfo.php');
+#include_once(dirname(__FILE__).'/../domain/Person.php');
+
+/**
+ * returns an array of  associative arrays of course evaluations based on the instructors name
+ * How to use data when called:
+ * index each evaluation by evaluations[x] ex: evaluations[1] will return the second evaluation
+ * index each data field by evaluations["key"] ex: evaluations["Topic"] will return the Topic of the evaluations
+ */
+function get_evalutaion_by_Instructor($InstructorName) {
```
Comment (MISTERTAC0H) [inline database/dbEvaluations.php:15]: Documentation for the function is good as well as the function itself. It works well and does exactly what youd expect it to based off of the name. Name of variables makes sense for its purpose.

File: CourseEval.php [hunk 2]
```diff
@@ -33,17 +33,47 @@
...
-            <option value="Sumit">Fair</option>  
-            <option value="Vineet">Poor</option>  
+            <option value="Select">Select</option>  
+            <option value="Excellent">Excellent</option>   
+            <option value="Good">Good</option>  
+            <option value="Fair">Fair</option>  
+            <option value="Poor">Poor</option>  
         </select>
+
+        <!-- code for check boxes (check all) -->
```
Comment (m-rose5) [inline CourseEval.php:48]: add a space between checkboxes to enhance readability

File: CourseEval.php [hunk 3]
```diff
@@ -32,11 +32,130 @@
...
-</html>
\ No newline at end of file
+</html>
+
+<?php
+    require_once('database/dbinfo.php');
+    include_once('domain/Person.php');
+    include_once('database/dbPersons.php');
+
+
```
Comment (m-rose5) [inline CourseEval.php:94]: variable names are clear to what data they hold

File: CourseEval.php [hunk 4]
```diff
@@ -59,19 +59,19 @@
         <label for="enthusiasm">Exhibited enthusiasm for the topic</label><br><br>
```
Comment (MISTERTAC0H) [inline CourseEval.php:60]: Changes are to variable names, made nonworking code work properly.

File: CourseEval.php [hunk 5]
```diff
@@ -32,11 +32,130 @@
         <?php require_once('header.php') ?>
```
Comment (m-rose5) [inline CourseEval.php:32]: include header comment with name of file and description of what the file does

File: CourseEval.php [hunk 6]
```diff
@@ -32,11 +32,130 @@
...
+            $responsiveToQuestions = in_array('Was responsive to questions', $opinions) ? "yes" : "no";
+            $enthusiasmForTopic = in_array('Exhibited enthusiasm for the topic', $opinions) ? "yes" : "no";
+        }
+
+        $increasedUnderstanding = $_POST['increasednderstanding'];
+        $learnedNewInfo = $_POST['learnedNewInfo'];
+        $improved = $_POST['improved'];
+        $interesting = $_POST['interesting'];
+
+        // Create database connection
```
Comment (m-rose5) [inline CourseEval.php:123]: comment what database it's connecting to and what table it will go to

File: database/dbEvaluations.php [hunk 7]
```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @version March 14, 2024
+ * @author Matthew Rose and Evan Guard
+ */
+include_once('dbinfo.php');
+#include_once(dirname(__FILE__).'/../domain/Person.php');
+
```
Comment (MISTERTAC0H) [inline database/dbEvaluations.php:8]: Overall documentation is good for each function and it describes what each function does.

File: CourseEval.php [hunk 8]
```diff
@@ -32,11 +32,130 @@
...
+                echo "Error: " . $sql . "<br>" . $conn->error;
+            }
+            // Close statement
+            $stmt->close();
+            
+        }
+
+        //Close connection
+        $conn->close();
+    }
```
Comment (m-rose5) [inline CourseEval.php:160]: Comments are good and describe what each block of code does

File: database/dbEvaluations.php [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+ * index each data field by evaluations["key"] ex: evaluations["Topic"] will return the Topic of the evaluations
+ */
+function get_evalutaion_by_Instructor($InstructorName) {
+    $con=connect();
+    $query = "SELECT * FROM dbEvaluations WHERE InstructorName = '" . $InstructorName . "'";
+    $result = mysqli_query($con,$query);
+    if (mysqli_num_rows($result) < 1) {
+        mysqli_close($con);
+        return false;
+    }
```
Comment (MISTERTAC0H) [inline database/dbEvaluations.php:22]: Comments describing what the step does would be good

**Pull request merged**